examples/ruby/start

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -e
+source ~/.rvm/scripts/rvm
+RUBY_VERSION="2.7.2"
+
+function announce() {
+  echo "++++++++++"
+  echo "$@"
+  echo "++++++++++"
+}
+
+if [[ ! -d "./out/ruby" ]]; then
+  announce "Client not found. Generating..."
+  ./bin/generate_client ruby > /dev/null
+  echo
+fi
+
+if [[ -z "$(docker-compose ps -q)" ]]; then
+  announce "Environment not found. Spinning up..."
+  ./bin/start_conjur > /dev/null
```

Comment:
I'm not sure if we should obfuscate _all_ the output for these commands. If something goes wrong, we want to still display an error message. I think we should use use `1>` instead of `>`, which will only redirect `stdout`, but not `stderr`.